src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ParameterExpander.java

```diff
@@ -249,6 +293,61 @@ private Integer getBuildStartedCodeReviewValue(Run r) {
         return map;
     }
 
+    /**
+     * Creates a list of the "standard" trigger parameters.
+     * They are present both for build started and completed.
+     * The parameters are:
+     * <ul>
+     * <li><strong>GERRIT_NAME</strong>: The Gerrit project name.</li>
+     * <li><strong>CHANGE_ID</strong>: The Gerrit change-id (SHA-1).</li>
+     * <li><strong>BRANCH</strong>: The branch of the project.</li>
+     * <li><strong>TOPIC</strong>: Topic name for change series.</li>
+     * <li><strong>CHANGE</strong>: The change number.</li>
+     * <li><strong>PATCHSET</strong>: The patchset number.</li>
+     * <li><strong>PATCHSET_REVISION</strong>: The patchset revision.</li>
+     * <li><strong>REFSPEC</strong>: The ref-spec. (refs/changes/xx/xxxx/z).</li>
+     * <li><strong>BUILDURL</strong>: The URL to the build.</li>
+     * <li><strong>VERIFIED</strong>: The verified vote.</li>
+     * <li><strong>CODE_REVIEW</strong>: The code review vote.</li>
+     * <li><strong>NOTIFICATION_LEVEL</strong>: The notification level.</li>
+     * </ul>
+     *
+     * @param r           the build.
+     * @param gerritEvent the event.
+     * @param labelVotes  all the labels added in Gerrit trigger config.
+     * @param notifyLevel the notify level.
+     * @return the parameters and their values.
+     */
+
+    private Map<String, String> createStandardParameters(Run r, GerritTriggeredEvent gerritEvent, Map<String, Integer> labelVotes, String notifyLevel) {
```

Comment:
Most of the logic is copied from the method above. Couldn't one of them use the other?

Reply:
I'll think of a clever way to reuse it.

Reply:
Done